community/io/src/main/java/org/neo4j/io/fs/DefaultFileSystemAbstraction.java

```diff
@@ -206,6 +206,12 @@ public long lastModifiedTime( File file )
         return file.lastModified();
     }
 
+    @Override
+    public void deleteFileOrThrow( File file ) throws IOException
```

Comment:
They have slightly different semantics wrt. to deleting files that don't exist, and I didn't want to uproot the use of deleteFile at this point because it has a lot of uses all over the code base. We'd have to examine each to see if they can tolerate the change in semantics.